src/components/personalBankAccountPropTypes.js

```diff
@@ -0,0 +1,55 @@
+import PropTypes from 'prop-types';
+
+export default PropTypes.shape({
+    /** General errors */
+    error: PropTypes.string,
+
+    /** Field specific errors */
+    errorFields: PropTypes.shape({
+        /** Plaid selector and password fields */
+        plaidSelector: PropTypes.shape({
```

Comment:
I thought we decided to remove this?

Reply:
Yes, you're right. I'm going to remove both that and errorfields.

Reply:
Err not removing errorFields since its being used by the validation